Solutions/Fortinet-FortiGate/Playbooks/Fortinet_ResponseOnIP/azuredeploy.json

```diff
@@ -45,11 +69,12 @@
 
   },
   "variables": {
-    "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('Playbook Name'))]",
-    "TeamsConnectionName": "[concat('teamsconnector-', parameters('Playbook Name'))]",
-    "FortinetConnectorConnectionName": "[concat('FortinetConnector-', parameters('Playbook Name'))]",
+    "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]",
+    "TeamsConnectionName": "[concat('teamsconnector-', parameters('PlaybookName'))]",
+    "FortinetConnectorConnectionName": "[concat('FortinetConnector-', parameters('PlaybookName'))]",
     "UserIdentifier": "[parameters('User Identifier Name')]",
-    "Functionappname": "[parameters('Function app Name')]"
+    "Functionappname": "[parameters('Function app Name')]",
+    "AzureManagementLink": "[concat('https://management','.azure','.com')]"
```

Comment:
@v-amolpatil , FortinetCustomConnectorName should be read from parameters instead of hardcoded at line#88 and line #4073